src/main/java/org/fcrepo/importexport/exporter/Exporter.java

```diff
@@ -353,26 +355,25 @@ private void exportDescription(final URI uri) throws FcrepoOperationFailedExcept
 
     }
 
-    private List<URI> filterInboundReferences(final URI uri, final Model model) {
-        final List<URI> inboundMembers = new ArrayList<>();
+    private Set<URI> filterInboundReferences(final URI uri, final Model model) {
+        final String withSlash = withSlash(uri).toString();
+        final String withoutSlash = withoutSlash(uri).toString();
+        final Set<URI> inboundMembers = new HashSet<>();
         final List<Statement> removeList = new ArrayList<>();
-        for (final String p : config.getPredicates()) {
-            final StmtIterator inbound = model.listStatements(null, createProperty(p), (RDFNode)null);
-            while (inbound.hasNext()) {
-                final Statement s = inbound.next();
-                final String subject = s.getSubject().toString();
-                if (!subject.startsWith(uri.toString())) {
-                    removeList.add(s);
-                    inboundMembers.add(URI.create(subject));
-                }
+        for (final StmtIterator inbound = model.listStatements(); inbound.hasNext(); ) {
+            final Statement s = inbound.next();
+            final String subject = s.getSubject().toString();
+            if (!subject.equals(withSlash) && !subject.equals(withoutSlash)) {
+                removeList.add(s);
+                inboundMembers.add(URI.create(subject));
```

Comment:
@escowles I am may not understand the inbound resources well but is there a case that other resource descriptions may include the same inbound resource reference or not? That is, more than one resource may contains the same inbound reference and when the method exportDescription(final URI uri, final URI binaryURI) is called repeatedly that inbound resource will be processed again and again.